openpype/pipeline/farm/patterning.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+import re
+
+
+def match_aov_pattern(host_name, aov_patterns, render_file_name):
+    """Matching against a `AOV` pattern in the render files.
+
+    In order to match the AOV name we must compare
+    against the render filename string that we are
+    grabbing the render filename string  from the collection
+    that we have grabbed from `exp_files`.
+
+    Args:
+        app (str): Host name.
+        aov_patterns (list): List of AOV patterns from AOV filters.
```

Comment:
aov_pattern isn't list, its dictionary:
```suggestion
        aov_patterns (dict):  AOV patterns from AOV filters.
```